### DISCUSSION TITLE
Getting error while running an existing modflow model using flopy package

### DISCUSSION THREAD

**Discussion by zargol7419** [modflowpy/flopy#1545] [Q&A] [answered]
I am using flopy package to change the content of well package and then to run the model and finally get all output heads. I have loaded my model successfully and I have changed the content of well package but when it comes to running the model after all these changes I've got this error. You know, there is no file named .name file in the file of my model but instead I have used ".mfn" file which can be considered as an alternative for ".name" file in my model. I don't know how can I solve this problem. Thanks in advance for your answers.
 [Here is my written code:](url)
![image](https://user-images.githubusercontent.com/103249900/190850123-e1d044e2-ad02-460e-b4db-47fc4f60ce8e.png)
[And here is the error:](URL)
![image](https://user-images.githubusercontent.com/103249900/190850213-eeb11515-9764-431c-9659-610c1c0b8043.png)

**Comment by wpbonelli** [comment, marked as answer]
Hi @zargol7419, are you sure namefile `step6.mfn` is in `model_ws` &mdash; models expect the namefile to exist in the workspace when `run_model` is called

If you want to load a model from one location, then switch to a new workspace and run the model there, you can use the following pattern:

```python
model = flopy.modflow.Modflow.load('my_namefile.nam', model_ws='my/workspace')
model.change_model_ws('new/workspace')
model.write_name_file()
model.write_input()
model.run_model()
```

**Reply by zargol7419** [reply]
Hi dear @w-bonelli actually I've got totally confused. Let me explain with more detail what I am exactly going to execute. Actually as I wanted to change the content of well package in the model I have changed HDF5 files to ASCII format by exporting native mf2k text so I have changed the content of well package as a txt file, and now I have to run this changed model to get the results. I can't understand what do you mean by switching to a new workplace. If it is possible please provide me with a more detailed answer. I really appreciate your patience.
here is a picture of my current workspace:
![image](https://user-images.githubusercontent.com/103249900/190901655-ca541010-3459-4a3d-b417-e3cfefc39d8e.png)

**Reply by wpbonelli** [reply]
It looks like you are using workspace `"something/../step6_MODFLOW/step6._ws"` as the `model_ws` argument. Are you sure this folder exists and contains your namefile? The `model_ws` must be a valid directory path, and the directory must contain the namefile provided as the first argument to `Modflow.load`.
